### ISSUE TITLE
debug 2fa auth issue for new users

### ISSUE THREAD

**Issue by lazzarello** [verygood-ops/verygood.openvpn#14]
new users created with the playbook may not have a local user password set for 2fa OpenVPN access. A suspicion is this could be a rule conflict with the SSH no passwords policy.

**Comment by lazzarello** [comment]
fixed in 2f55824

**Comment by lazzarello** [comment]
confirmed works in dev

```
Tue Sep 13 21:21:41 2016 192.168.34.1:49425 TLS: Initial packet from [AF_INET]192.168.34.1:49425, sid=9b5887ea bc3fc41f
Tue Sep 13 21:21:41 2016 192.168.34.1:49425 VERIFY OK: depth=1, CN=OpenVPN-CA
Tue Sep 13 21:21:41 2016 192.168.34.1:49425 VERIFY OK: depth=0, CN=OpenVPN-Client-lee-test
Tue Sep 13 21:21:41 2016 192.168.34.1:49425 PLUGIN_CALL: POST /usr/lib/openvpn/openvpn-plugin-auth-pam.so/PLUGIN_AUTH_USER_PASS_VERIFY status=0
Tue Sep 13 21:21:41 2016 192.168.34.1:49425 TLS: Username/Password authentication succeeded for username 'vagrant' [CN SET]
Tue Sep 13 21:21:41 2016 192.168.34.1:49425 Data Channel Encrypt: Cipher 'AES-256-CBC' initialized with 256 bit key
Tue Sep 13 21:21:41 2016 192.168.34.1:49425 Data Channel Encrypt: Using 160 bit message hash 'SHA1' for HMAC authentication
Tue Sep 13 21:21:41 2016 192.168.34.1:49425 Data Channel Decrypt: Cipher 'AES-256-CBC' initialized with 256 bit key
Tue Sep 13 21:21:41 2016 192.168.34.1:49425 Data Channel Decrypt: Using 160 bit message hash 'SHA1' for HMAC authentication
Tue Sep 13 21:21:41 2016 192.168.34.1:49425 Control Channel: TLSv1, cipher TLSv1/SSLv3 DHE-RSA-AES256-SHA, 4096 bit RSA
Tue Sep 13 21:21:41 2016 192.168.34.1:49425 [vagrant] Peer Connection Initiated with [AF_INET]192.168.34.1:49425
Tue Sep 13 21:21:41 2016 vagrant/192.168.34.1:49425 MULTI_sva: pool returned IPv4=10.0.2.134, IPv6=(Not enabled)
Tue Sep 13 21:21:41 2016 vagrant/192.168.34.1:49425 MULTI: Learn: 10.0.2.134 -> vagrant/192.168.34.1:49425
Tue Sep 13 21:21:41 2016 vagrant/192.168.34.1:49425 MULTI: primary virtual IP for vagrant/192.168.34.1:49425: 10.0.2.134
Tue Sep 13 21:21:43 2016 vagrant/192.168.34.1:49425 PUSH: Received control message: 'PUSH_REQUEST'
Tue Sep 13 21:21:43 2016 vagrant/192.168.34.1:49425 send_push_reply(): safe_cap=940
Tue Sep 13 21:21:43 2016 vagrant/192.168.34.1:49425 SENT CONTROL [vagrant]: 'PUSH_REPLY,route 10.0.0.0 255.255.0.0,route 10.0.2.129,topology net30,ping 5,ping-restart 30,ifconfig 10.0.2.134 10.0.2.133' (status=1)
```
